apps/nrm/models/submission.js

```diff
@@ -199,8 +199,25 @@ module.exports = (data, token) => {
   response.PVFamily = data['family'];
   response.PVEducation = data['education'];
   response.PVEmploymentHistory = data['employment-history'];
-  response.ExploitationDates = data['when-did-the-exploitation-take-place'];
-  response.MultipleExploitationSituations = data['more-than-one-exploitation-situation'];
+
+  const moreSituations = {
+    yes: 'Yes',
+    no: 'No',
+    'i-dont-know': 'DontKnow'
+  };
+
+  if(data['potential-victim-exploitative-situation-multiple']) {
+    response.MoreThanOneExploitationSituations =
+      moreSituations[data['potential-victim-exploitative-situation-multiple']];
+  } else {
+    response.MultipleExploitationSituations = data['more-than-one-exploitation-situation'];
+  }
```

Comment:
I don't think we need the if/else here - response.MultipleExploitationSituations is legacy data so it should always go to icasework if it was completed on the form, even if the user has since answered response.MoreThanOneExploitationSituations. I would reinstate response.MultipleExploitationSituations = moreSituations[data['potential-victim-exploitative-situation-multiple']];  as it was before and then add  response.MoreThanOneExploitationSituations = moreSituations[data['potential-victim-exploitative-situation-multiple']]; on the line before/after